nixos/modules/security/hidepid.nix

```diff
@@ -33,6 +33,12 @@ in
   };
 
   config = mkIf cfg.enable {
+    assertions =
+     [ { assertion = cfg.procGroup != null -> (config.users.groups."${cfg.procGroup}" or "") != "";
+         message = "the group named at security.hideProcessInformation.procGroup does not exist!";
+       }
+     ];
+
```

Comment:
This would not necessarily be so -- `mutableUsers` is on by default so users can add their own groups circumventing NixOS configuration.